WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/LayoutPickerStoryboard.storyboard

```diff
@@ -125,7 +125,7 @@
                         </constraints>
                     </view>
                     <navigationItem key="navigationItem" title="Preview" id="yl7-1R-9yC">
-                        <barButtonItem key="rightBarButtonItem" style="plain" id="a9M-Fe-sUg">
```

Comment:
This file has gone away now and split became [LayoutPreviewViewController.xib](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Gutenberg/Layout%20Picker/LayoutPreviewViewController.xib). This button is now being added programmatically so we should be able to discard this change.

Reply:
Thanks for spotting this @chipsnyder ! I just merged in `develop` so this change should be gone now.